Makefile

```diff
@@ -761,9 +769,18 @@ test-e2e-cluster:
 	$(GO) test ./... -race -tags e2ecluster
 
 stress-frontend-%: build/toolchain/python/
-	$(TOOLCHAIN_DIR)/python/bin/locust -f $(REPOSITORY_ROOT)/test/stress/frontend.py --host=http://localhost:51504 \
+	$(TOOLCHAIN_DIR)/python/bin/locust -f $(REPOSITORY_ROOT)/test/stress/frontend.py --host=http://localhost:$(FRONTEND_PORT) \
 		--no-web -c $* -r 100 -t10m --csv=test/stress/stress_user$*
 
+external-ip-locust-master: external-host-locust-master
+	@echo :$(LOCUST_PORT)
+
+external-ip-om-frontend: external-host-om-frontend
+	@echo :$(FRONTEND_PORT)
+
+external-host-%:
```

Comment:
This target isn't useful there's other patterns to use here. Like proxy or exposing via LoadBalancer.